build_and_deploy/clients/artifacts_client.ts

```diff
@@ -424,7 +424,6 @@ export class ArtifactClient {
                 return `https://dev.azuresynapse.azure.cn`;
             case Env.usnat.toString():
                 return `https://dev.azuresynapse.usgovcloudapi.net`;
-            case Env.blackforest.toString():
             default:
                 throw new Error('Environment validation failed');
```

Comment:
Could you add more to this comment, so users can figure our what they missed, what the available values are etc?